unitpackage/entry.py

```diff
@@ -832,9 +863,13 @@ def save(self, *, outdir, basename=None):
             True
 
         When a ``basename`` is set, the files are named ``basename.csv`` and ``basename.json``.
-        Note that for a valid frictionless Data Package this base name
-        MUST be lower-case and contain only alphanumeric
-        characters along with ".", "_" or "-" characters'::
+
+        .. note::
+            For a valid frictionless Data Package the basename
+            MUST be lower-case and contain only alphanumeric
+            characters along with ``.``, ``_`` or ``-`` characters'
```

Comment:
Should we rely on the handling of frictionless?

https://github.com/frictionlessdata/frictionless-py/blob/main/frictionless/vendors/stringcase.py

Reply:
afaik does frictionless enforce always lowercase. The validity of names is checked upon loading a datapackage by frictionless, so we do not run our test. I added the note here so people don't have to dig deep into the frictionless documentation. If the frictionless policy changes, we might have to rewind the change from this PR.

I think adopting a lowercase solution is the most simple one. Creating, for example, camel case strings is not trivial since the meaning of the string is often unclear. Assume your string is `CVmeasurement` then that should result in `cvMeasurement`, which can not be achieved unless you provide an appropriate dictionary.

Reply:
We might want to adopt camel case in our JSON files.

According to the [getting started section](https://github.com/frictionlessdata/frictionless-py/blob/95098b3794fc6cb5380e13d3e1ee128100b525ae/docs/getting-started.md?plain=1#L59) from frictionless
```markdown
Arguments conform to the following naming convention:
- for Python interfaces, they are snake_cased, e.g. `missing_values`
- within dictionaries or JSON objects, they are camelCased, e.g. `missingValues`
- in the command line, they use dashes, e.g. `--missing-values`
```